autoload/fzf_mru/actions.vim

```diff
@@ -21,6 +21,14 @@ function! fzf_mru#actions#options() abort
   return options
 endfunction
 
+function! s:p(...)
```

Comment:
Can you change the name of the function for something meaningful like `s:preview`. I know most of the code in this project is using one letter names, especially code copied from `ctrlp` but I would like to avoid adding more to it.

Reply:
I've fixed this and the above comment(s). Will push shortly.